Source/ModelGroup.cs

```diff
@@ -16,6 +16,7 @@ internal abstract partial class ModelGroup
         public List<Runtime.Image> UsedFigures = new List<Runtime.Image>();
         public int Id = -1;
         public bool NoSampleImages = false;
+        public bool ApplyPostRuntimeChanges = false;
```

Comment:
Why do you need this?

Reply:
This was being used to prevent running `PostRuntimeChanges()` for every model group, but I forgot that it doesn't do anything unless a model group overrides it. Not needed, I've removed the bool.